addons/website/views/website_templates.xml

```diff
@@ -326,6 +326,28 @@
     </xpath>
 </template>
 
+<!-- Navbar templates -->
+<template id="navbar_showcase" inherit_id="website.layout" name="Navbar Showcase" active="False">
```

Comment:
Prefix the id by "template_" for your two templates, it will be clearer for the future